api/proto/index/v2/index.proto

```diff
@@ -0,0 +1,164 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+package index.v2;
+option go_package = "./index/v2;index";
+
+// Service provides an external API for managing the index records
+service Service {
```

Comment:
In case of renaming the document in the customer system, should we also allow renaming in Simila (changing the path)?

Reply:
absolutely, this is the idea. I just skip it to avoid the mess. We will add it later